Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <PropertyGroup>
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
Should this also have the property for signed packages? (`SignAssembly` IIRC?)

Reply:
I believe that is on by default with arcade.